src/components/widgets/PickListPopup/PickListPopup.tsx

```diff
@@ -69,7 +72,24 @@ export const PickListPopup: FunctionComponent<PickListPopupProps & PickListPopup
         <div>
             <h2 className={styles.title}>{props.widget.title}</h2>
             {(props.bcLoading)
-                ? <Skeleton loading paragraph={{rows: 5}} />
+            ? <Skeleton loading paragraph={{rows: 5}} />
+            : (props.widget.options
```

Comment:
Decision on a specific hierarchy type should be encapsulated by `<Table />`, there is no need for a PickListPopup to know them.